src/model/client.cc

```diff
@@ -85,7 +85,7 @@ Json::Value Client::SyncMetadata() const {
 Json::Value Client::SyncPayload() const {
     Json::Value result;
     if (NeedsPOST()) {
-        result["id"] = Json::Int64(ID());
+        result["id"] = Json::Int64(-LocalID());
```

Comment:
is "client_assigned_id" field still needed if "id" field has the same value? can you show the Sync spec which refers to this? [this one](https://github.com/toggl/toggl-api-clients/tree/master/Sync%20Specification) seems to not go into details of how JSONs should look

Reply:
I'll answer this on Slack